src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -190,7 +194,10 @@
       </tr>
       {% endif %}
       <tr>
-        <td>Address:</td>
+        <td class="keep-formatting">Address:
+          {% if defendant.addressAmended %}
+          (as amended)
```

Comment:
is this where its supposed to go? will this break robotics?
we used to have it after this address first line I think

Reply:
That's what Marc James and Art would like

Reply:
will it break robotics?

Reply:
I'm not sure Tim, will need to check

Reply:
@rattur I managed to find previous version. Here is what we had before https://github.com/hmcts/cmc-claim-store/pull/279/files#diff-5b3cba7f1066682c02d7992300655a03L175.

@singh-surjeet removed that feature on 22nd Match.

I wonder whether bringing previous implementation would make it easier for RPA team. I guess we need to wait for response from them.

Reply:
This new version is preferred from the previous implementation, therefore, had conversations and sent emails with the new pdf attached to Clem and RPA team. Will wait for the outcome